health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/CreateChecklist.js

```
@@ -0,0 +1,647 @@
+                style={{ marginTop: "2rem", marginBottom: "2rem" }}
+                // icon={<AddIcon style={{ height: "1.5rem", width: "1.5rem" }} fill={PRIMARY_COLOR} />}
+                onClick={popShow}
+              />
+            </div>
+          </div>
+          {showPopUp && (
+            <PopUp
+              className={"custom-pop-up"}
+              type={"default"}
+              heading={t("CHECKLIST_PREVIEW")}
+              children={[
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)